ArkEcosystem/laravel-foundation
# feat: table compact styling

## Summary

- Adds the compact styling to the table 
- Update docs to include table usage including this new feature

### To test:

https://app.clickup.com/t/1rr57f4

1. Add this PR to the [msq one](https://github.com/ArkEcosystem/marketsquare.io/pull/2456) and [explorer one](https://github.com/ArkEcosystem/explorer/pull/1011)
2. See if the table really applies the styling

## Checklist
